src/packages/client/src/generation/TSClient/helpers.ts

```diff
@@ -87,6 +87,8 @@ export function getArgFieldJSDoc(
     const comment = JSDocs[action]?.fields[fieldName](singular, plural)
     return comment as string
   }
+
+  return undefined
```

Comment:
Curious to know why this was needed?

Reply:
Maybe because I see ` "noImplicitReturns": true,` added to tsconfig

Reply:
@millsp Personal preference